package.json

```diff
@@ -8,6 +8,7 @@
     "start": "next start",
     "lint": "next lint",
     "occurances:ingest": "npx ts-node ./src/scripts/occurances/ingest.ts",
+    "occurrances:ingest-prod": "dotenv -e .env.prod -- npx ts-node ./src/scripts/occurances/ingest.ts",
```

Comment:
integrate dotenv into pnpm because there is a collision betwen ruby dotenv and npm dotenv when installed globally

Reply:
I'll take a look. Ironically the first thing I did was test a few things on armory 😂

I may just decide to redo everything in nest tho 🐦‍🔥 I'll take a look for your link